packages/manager/apps/hub/src/dashboard/products-list/cloud-connect.routing.js

```diff
@@ -0,0 +1,53 @@
+import { compact, get, map, set } from 'lodash-es';
+
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+import { component, resolves } from './config';
+import { POP_MAP } from './products-list.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dashboard.ovh_cloud_connect', {
```

Comment:
I'm not sure we should add a state for this as with solarsteein we will need to duplicate this configuration, maybe instead we should find a way to fetch/provide a configuration depending on the product that we can pass (and avoid duplicating states) because any change we will have to make to this configuration we will have to replicate it elsewhere